src/components/Routes.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import SearchForm from './SearchForm';
+
+//this component will allow route params in the url
+const Routes = () => {
+  return (
+    <Switch>
+      <Route path="/:params" component={SearchForm} />
```

Comment:
Right now this param is very permissive, ex if you type "../foo" it will still render the `SearchForm` component. Some experimenting with the params to make them stricter would be 👍

Reply:
Fixed to render a 404 error page.